ql/src/test/queries/clientpositive/udf_nullif.q

```diff
@@ -27,3 +27,12 @@ select	nullif(a,b),
 	nullif(b,c),
 	nullif(c,d),
 	nullif(d,a) from t0;
+
+SELECT assert_true(NULLIF(array(1,2,3),array(1,2,3)) is null);
+SELECT assert_true(NULLIF(array(1,2,3),array(3,2,1)) is not null);
+
+SELECT assert_true(NULLIF(named_struct("c", 1),named_struct("c", 1)) is null);
+SELECT assert_true(NULLIF(named_struct("c", 1),named_struct("c", 2)) is not null);
+
+SELECT assert_true(NULLIF(map('a',1,'b',2),map('a',1,'b',2)) is null);
+SELECT assert_true(NULLIF(map('a',1,'b',2),map('a',1,'b',3)) is not null);
```

Comment:
I haven't used `assert_true` before so I don't know what is the additional benefit it brings here. Can't we simply compare with the `.q.out` that the result of NULLIF is the expected one? I guess this would make also the use of `is null/is not null` redundant since we could see directly the result in the `.q.out` file.